src/pyasm/web/cherrypy30_startup.py

```diff
@@ -449,7 +506,100 @@ def CORS():
 
 
 
-    def register_project(self, project, config, site=None):
+    def register_project(self, project, config, site=None, need_upgrade=[False]):
+
+
+        if Config.get_value("portal", "auto_upgrade") == 'true':
+
+
+            if site and site != 'default':
```

Comment:
remove this if statement. We want to upgrade all sites including the default site.

site can be = ""